pcawg_metadata_parser/generate_ucsc_pilot_donors_json.py

```diff
@@ -271,6 +394,16 @@ def main(argv=None):
 
     donor_fh = open(metadata_dir+'/ucsc_polit_donor_'+es_index_reorganize+'.jsonl', 'w')
 
+    pilot_tsv_fh = open(metadata_dir + '/ucsc_polit_donor_' + es_index_reorganize + '.tsv', 'w')
+    
+    # read the tsv fields file and write to the pilot donor tsv file
+    tsv_fields = 'ucsc_pilot_donor_tsv_fields.txt'
+    with open(tsv_fields, 'r') as t:
+        for line in t:
+            pilot_tsv_fh.write(line.rstrip() + '\t')
```

Comment:
This will leave an extra '\t' at the end. We can build an array first, then join the array elements with '\t' to build a string, after that output the string and add a '\n'.